components/support/error/src/lib.rs

```diff
@@ -1,79 +1,77 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
 /// Define a wrapper around the the provided ErrorKind type.
 /// See also `define_error` which is more likely to be what you want.
+
 #[macro_export]
 macro_rules! define_error_wrapper {
     ($Kind:ty) => {
-        /// Re-exported, so that using crate::error::* gives you the .context()
-        /// method, which we don't use much but should *really* use more.
-        pub use failure::ResultExt;
-
         pub type Result<T, E = Error> = std::result::Result<T, E>;
+        use std::fmt;
```

Comment:
It's not good to have misc `use` in item-level macros, since it brings things to scope unexpectedly. The old code had `pub use failure::ResultExt`, but that's present in order to expose some useful extension trait methods.